### ISSUE TITLE
[Bug]: The query results of the two versions are abnormal

### ISSUE THREAD

**Issue by StevenSama** [milvus-io/milvus#17378]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:milvus master-dev（docker pull milvusdb/milvus-dev:master-20220602-48cf63a2） vs 2.0.2
- Deployment mode(standalone or cluster):standalone 
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): Ubuntu
- CPU/Memory: 24C/32G
- GPU: 
- Others:
```

### Current Behavior

![image](https://user-images.githubusercontent.com/6235700/172110960-6fb8aa03-a172-43df-9b1f-77b32dbd254c.png)
figure 1
![image](https://user-images.githubusercontent.com/6235700/172111081-51d5de56-0b23-4c7d-941c-aa59176dc893.png)
figure 2

Figure 1 is the query result of version 2.0.2, and Figure 2 is the query result of the master-dev branch. The sets of the two versions are the same, and the queried vectors are in the library

### Expected Behavior

The query results should be consistent

### Steps To Reproduce

```markdown
1. Create a new collection on 2.0.2 and insert 74 768-dimensional vectors
2. Create a new collection in the master-dev version and insert the same vector as step 1
3. Querying two different versions of the collection respectively shows that the results are very different, thinking that the results of the latest version are wrong
```

**Comment by StevenSama** [comment]
useing docker image，docker pull milvusdb/milvus-dev:master-20220602-48cf63a2

**Comment by yanliang567** [comment]
@StevenSama did you try with the milvus sdk? I am asking because I want to check whether it is an Attu issue or Milvus issue?

/assign @StevenSama 
/unassign

**Comment by StevenSama** [comment]
> 

It has nothing to do with the SDK, the result returned by using the SDK is also wrong
R{status=0, data=status {
}
results {
  num_queries: 1
  top_k: 10
  fields_data {
    type: Int64
    field_name: "id"
    scalars {
      long_data {
        data: 22087
        data: 167961
        data: 27381
        data: 384765
        data: 354327
        data: 349283
        data: 29769
        data: 342729
        data: 332775
        data: 367571
      }
    }
    field_id: 100
  }
  scores: -1.000699
  scores: -0.79514015
  scores: -0.78901744
  scores: -0.77581424
  scores: -0.77057266
  scores: -0.7677765
  scores: -0.76437044
  scores: -0.76239073
  scores: -0.7618445
  scores: -0.76022065
  ids {
    int_id {
      data: 22087
      data: 167961
      data: 27381
      data: 384765
      data: 354327
      data: 349283
      data: 29769
      data: 342729
      data: 332775
      data: 367571
    }
  }
  topks: 10
}

IVF_SQ8 256 IP
nprobe 10 IP topK 10 

The searched vector is in the library

**Comment by StevenSama** [comment]
@yanliang567

**Comment by yanliang567** [comment]
@yhmo  could you please take a look here?
/assign @yhmo 
/unassign @StevenSama

**Comment by longjiquan** [comment]
@StevenSama thx for reporting the so detailed information. This issue will be fixed by #17590 .
/assign @StevenSama
